nuget/microsoft.cppwinrt.nuspec

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>microsoft.cppwinrt</id>
+    <version>0.0.0</version>
```

Comment:
http://docs.nuget.org/ndocs/create-packages/prerelease-packages#semantic-versioning
Nothing in this delta is dependent on the version number you end up choosing though.